Cargo.lock

```diff
@@ -540,6 +540,7 @@ dependencies = [
  "codechain-json",
  "codechain-key",
  "primitives",
+ "rand 0.6.1",
```

Comment:
We should be careful when we add a dependency in Foundry.
Please discuss with me before adding a new package in the Foundry.

Reply:
Ok I'll be careful next time

Reply:
I think they're quite similar; Use `rand`, with constant seed and iterate.

Reply:
Thanks. Let's use it.